pgjdbc/src/main/java/org/postgresql/jdbc/PgSQLXML.java

```diff
@@ -72,10 +72,11 @@ public PgSQLXML(BaseConnection conn, @Nullable String data) {
 
   private PgSQLXML(BaseConnection conn, @Nullable String data, boolean initialized) {
     this.conn = conn;
-    this.data = data;
+    this.value = data;
     this.initialized = initialized;
     this.active = false;
     this.freed = false;
+    setType("xml");
```

Comment:
change this to this.type="xml" to fix the checker, no need to call the function here.

Reply:
Done, thank you.